flight/targets/pipxtreme/fw/pios_board.c

```diff
@@ -164,43 +323,34 @@ void PIOS_Board_Init(void) {
 				}
 
 #endif /* PIOS_INCLUDE_PPM */
-				break;
-			}
-		case HWTAULINK_PPMPORT_DISABLED:
-		default:
-			break;
-	}
+        break;
+    }
+    case HWTAULINK_PPMPORT_SPORT:
```

Comment:
I think the shared system could do it (we don't care what the target module is.  But-- right now we don't really handle PPM in any way and I'd likely have a different wrapper call for receiver-port type things that have serial etc on them.